Example/Core/Tests/FIROptionsTest.m

```diff
@@ -584,4 +585,16 @@ - (void)testVersionConsistency {
   XCTAssertEqualObjects(str, [NSString stringWithUTF8String:(const char *)FIRCoreVersionString]);
 }
 
+// Repeat test with more Objective C.
+- (void)testVersionConsistency2 {
+  NSRange major = NSMakeRange(0, 1);
+  NSRange minor = NSMakeRange(1, 2);
+  NSRange patch = NSMakeRange(3, 2);
+  NSString *str =
+      [NSString stringWithFormat:@"%@.%d.%d", [kFIRLibraryVersionID substringWithRange:major],
```

Comment:
Is there a reason we're not doing `intValue` here as well? I see it was from below but not sure why that was the case.

Reply:
There's no leading zero on the major.